lib/sidekiq/statistic/statistic/workers.rb

```diff
@@ -17,6 +17,17 @@ def display
         end
       end
 
+      def display_by_last_status
+        {
+          passed: filter_last_job_status('passed'),
+          failed: filter_last_job_status('failed')
+        }
+      end
+
+      def filter_last_job_status(status)
+        display.select { |worker| worker[:last_job_status] == status }
```

Comment:
Can u try to feed a huge number of data into Redis just to allow us to identify performance issues here?